src/Stormpath/Cache/MemoryCacheManager.php

```diff
@@ -0,0 +1,10 @@
+<?php namespace Stormpath\Cache;
+
+
+class MemoryCacheManager implements CacheManager {
+
+    public function getCache()
+    {
+        return new MemoryCache();
```

Comment:
Going to talk about that at our 2pm

On Wed Jan 28 2015 at 1:46:20 PM ecrisostomo <notifications@github.com>
wrote:

> In src/Stormpath/Cache/MemoryCacheManager.php
> <https://github.com/stormpath/stormpath-sdk-php/pull/18#discussion_r23711158>
> :
>
> > @@ -0,0 +1,10 @@
> > +<?php namespace Stormpath\Cache;
> > +
> > +
> > +class MemoryCacheManager implements CacheManager {
> > +
> > +    public function getCache()
> > +    {
> > +        return new MemoryCache();
>
> What about getting the Cache instance from the cache String key (i.e.:
> from the region)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/stormpath/stormpath-sdk-php/pull/18/files#r23711158>.
>